### ISSUE TITLE
Implement JDBC driver

### ISSUE THREAD

**Issue by andygrove** [ballista-compute/ballista#83]
As a user, I want to be able to connect to a Ballista cluster from JDBC and run queries. Note that there is already PoC code for this under `jvm/jdbc`.

**Comment by stspyder** [comment]
Is somebody working on this? If not, I'd like to take this.

**Comment by andygrove** [comment]
Hi @stspyder I don't think anyone is working on this. There is a POC that was working at one point under jvm/jdbc directory. We should probably work together to plan out some more detailed stories around what needs to be done. 

A good first step would be adding JDBC to the integration tests perhaps?

**Comment by stspyder** [comment]
Yeah makes sense. I did take a look at the existing POC. I guess it would need to be updated, because I noticed that it sends the sql as string to the executor instead of a serialized logical plan. As I get it to work I'll also write a simple integration test (like what we have for rust right now).

**Comment by andygrove** [comment]
That sounds good. Also take a look at https://github.com/ballista-compute/ballista/issues/262 since it is related. It would be nice to be able to send SQL to a scheduler and get the logical plan back, or just have the scheduler take care of executing the SQL.

**Comment by stspyder** [comment]
OK. Thank You. I'll look into that too. I'll post any questions that I may have in #262 's thread.
